featuretools/primitives/standard/datetime_transform_primitives.py

```diff
@@ -515,6 +515,53 @@ def month(vals):
         return month
 
 
+class PartOfDay(TransformPrimitive):
```

Comment:
Sorry, just thought of this, but can you add boundaries of all different types in the doc string?